### DISCUSSION TITLE
Spoilers lever animation when bound to an axis

### DISCUSSION THREAD

**Discussion by BobbyFuzzy** [flybywiresim/aircraft#2995] [Ideas]
I wonder if the spoilers lever animation when bound to an axis on a controller can be improved. At the moment when it is bound to an axis it appears to move in stages with some lag so makes it difficult to precisely move.

The lever seems to move nice and smooth when clicking and dragging.

**Comment by Inaxair** [comment]
Hello @BobbyFuzzy 

Do you have the TCA pack?

Best regards,
Inaxair from FlyByWire Simulations

**Reply by BobbyFuzzy** [reply]
Yes minus the add-on. Still waiting for that. I had spoilers mapped to the slider on the sidestick.
